src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -95,6 +95,10 @@
  *  and verify the expected HTTP requests have been sent to the mocked server.
  * @todo #69:30min Fix NullPointerExceptions in CL build for Open JDK6.
  * @todo #69:30min Fix UnsupportedClassVersionError in CL build for Open JDK6.
+ * @todo #82:30min Clean unnecessary docker setup (src/docler/Dockerfile),
+ *  make the configuration minimal. Set versions to the used packages
+ *  (e.g. lock java version instead of default jdk). Discuss with architect
```

Comment:
@amihaiemil please also add an info that after changes it needs to be uploaded to Docker Hub and built there